hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -13,6 +13,160 @@
                      "hw/ip/tlul/data/tlul_testplan.hjson"]
 
   testpoints: [
+
+    //////////////////
+    // Host Device  //
+    // RV_CORE_IBEX //
+    //////////////////
+    //
+    // Note: the Debug Module will be referred to as RVDM.
+    {
+      name: chip_ibex_debug_basic
+      desc: '''Verify correct operation of rv_core_ibex while in debug mode.
+
+            Trigger the RVDM to send a debug request to rv_core_ibex, which should then jump to the
+            debug ROM and execute the code there before returning via `dret` and executing the rest
+            of the main program.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_ibex_debug_single_step_test
+      desc: '''Verify correct operation of rv_core_ibex while single-step debugging.
+
+            Set `dcsr.step` in the core.
+            Trigger the RVDM to send a debug request to rv_core_ibex, which should then jump to the
+            debug ROM and start code execution.
+            Write `dmcontrol.resumereq`, and check that the core jumps out of debug mode, executes
+            one instruction, then jumps back into debug mode.
+            Do this a random number of times before returning via `dret`and executing the rest of
+            the main program.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_ibex_debug_ebreak_test
+      desc: '''Verify functionality of EBREAK logic at the system level.
+
+            Trigger the RVDM to send a debug request to rv_core_ibex, which should then jump to the
+            debug ROM and start code execution.
+            Insert `ebreak` instruction into the debug ROM, and check that the hart completely
+            halts, and restarts execution of the debug ROM.
+
+            TODO: similar to block level test, do we need this feature tested again?
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_ibex_interrupt_test
+      desc: '''Verify correct operation of rv_core_ibex when taking interrupts from PLIC.
+
+            Trigger random interrupts from the PLIC, and check that the core takes the correct
+            interrupt, jumps to the proper handler, and returns to finish execution of the main
+            program.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_ibex_recoverable_nmi_test
+      desc: '''Verify correct operation of rv_core_ibex recoverable NMI functionality.
+
+            TODO: Greg to check with SW team if this mechanism still is sensible.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_ibex_mixed_use_sw_test
```

Comment:
There are a couple of tests related to the A/B slots in the ROM testplan, (e.g. [rom_e2e_boot_policy_newer](https://github.com/lowRISC/opentitan/blob/a00ce54bf6341a296c0d3864008487e0b65e5787/sw/device/silicon_creator/rom/data/rom_testplan.hjson#L498)). I don't think another testpoint will be needed for this.

Reply:
No action needed - resolving.